hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -107,28 +107,44 @@
     {
       name: pattgen_op_cg
       desc: '''
-            The following coverage strategy is implemented:
-            * The counters are covered at the max
-              and min corner cases and for the middle values
-            * Data bus has functional coverage using walking ones and zeroes
-            * One bit width variables are wrapped in one cover point
-              while counters and data have designated cover points
-            * Channel enable signal is stand alone cover point
+            Covers the interrupt done and event done for channel 0 and 1.
             '''
     }
     {
       name: roll_cg
       desc: '''
-            Covers that all the counters revert to zero when they reach
-            the maximum value.
-            Individual initial and maximum counter values
+            Covers the individual initial and maximum counter values
```

Comment:
I could not search for the name `roll_cg`, is that a different name in DV code or not implemented yet?